src/components/Nav.css

```diff
@@ -0,0 +1,53 @@
+.main__heading__section {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: var(--white);
+  padding: 10px 5%;
+}
+
+.menu {
+  list-style-type: none;
+  display: flex;
+  gap: 15px;
+  padding-top: 10px;
+}
+
+.link-items {
+  font-size: 18px;
+  color: var(--ash);
+}
+
+a {
+  color: var(--ash);
+}
+
+a:hover {
+  text-decoration: none;
+}
+
+a:hover,
+.btn__primary:hover {
+  text-decoration: 1px underline var(--blue);
+  transition: var(--ease);
+}
+
+.box__headings {
+  display: flex;
+  gap: 20px;
+}
+
+.logo {
+  color: var(--blue);
+  font-size: 35px;
+}
+
+.box__heading3 {
+  font-weight: var(--weight);
+}
+
+.account {
+  color: var(--blue);
+  font-size: 30px;
+  cursor: pointer;
+}
```

Comment:
- [ ] Please kindly make sure your design sticks to the original design, (eg: font, margin, item alignment, color,...)

Your design
<img width="936" alt="Screen Shot 2022-12-23 at 00 11 10" src="https://user-images.githubusercontent.com/74501523/209226717-3c5d50dc-8aab-4a9c-a015-8ad2f1b9478d.png">

The original design
<img width="906" alt="Screen Shot 2022-12-22 at 23 38 50" src="https://user-images.githubusercontent.com/74501523/209226496-cc4ebbf9-d31a-4185-8636-ffd3d31dc40e.png">